docker-compose.yml

```diff
@@ -78,20 +78,18 @@ nginx:
 
 # Consul is the service catalog that helps discovery between the components
 consul:
-  image: progrium/consul:latest
-  command: -server -bootstrap -ui-dir /ui
+  image: autopilotpattern/consul:latest
+  command: >
+    /usr/local/bin/containerpilot
+    /bin/consul agent -server
+    -bootstrap-expect 1
```

Comment:
I don't think that works -- when a node comes up if it expects only 1 and the CNS name it resolves is itself, then it'll form a split brain cluter.